Makefile

```diff
@@ -33,10 +33,19 @@ build-runtime:
 	mkdir -p $(BINDIR)
 	GOARCH=$(RUNTIME_ARCH) GOOS=$(RUNTIME_PLATFORM) go build -ldflags '$(LDFLAGS)' -o $(BINDIR)/$(MIXIN)-runtime$(FILE_EXT) ./cmd/$(MIXIN)
 
-build-client:
+build-client: build-templates
 	mkdir -p $(BINDIR)
 	go build -ldflags '$(LDFLAGS)' -o $(BINDIR)/$(MIXIN)$(FILE_EXT) ./cmd/$(MIXIN)
 
+build-templates: get-deps
```

Comment:
@vdice Just wanted you to know that I'm stealing your code. 🙇‍♀️

Reply:
I'm still running up a large deficit in the reverse, so every little bit helps :)